templates/certified/model-details.html

```diff
@@ -8,322 +8,344 @@
-                  <p>{{ release.bios }}</p>
-                {% endif %}
-                {% if release.notes %}
-                  <h4 class="u-no-margin--bottom">Notes</h4>
-                  {% for note in release.notes %}
-                    <p>
-                      {{ note.comment | replace('\n', '<br />') | safe }}
-                    </p>
-                  {% endfor %}
-                {% endif %}
-                  <h4>Hardware</h4>
-                  <table aria-label="Hardware" class="u-no-margin--bottom">
-                    <tbody>
-                      {% for hardware_subtitle, values in release["components"].items() %}
-                        <tr>
-                          <th colspan="2" class="u-text--muted">{{ hardware_subtitle }}</th>
-                          <td colspan="8">
-                            {% for value in values %}
-                              <p style="margin-bottom: 0.5rem;">
-                                {{ value.name }}
-                                {% if value.bus in ["usb", "pci"] %}
-                                  {{ value.bus }} ({{ value.identifier }}
-                                  {% if value.subsystem != '' %}{{ value.subsystem }}{% endif %}
-                                  )
-                                {% endif %}
-                              </p>
-                            {% endfor %}
-                          </td>
-                        </tr>
-                      {% endfor %}
-                    </tbody>
-                  </table>
-                  <p class="u-sv3">
-                    <a href="/certified/{{ canonical_id }}/{{ release.version }}">Hardware details&nbsp;&rsaquo;</a>
-                  </p>
-                </section>
-              </li>
+  </section>
+  <div class="row">
+    {% if release_details["releases"]|length > 1 %}
+    <div class="row" style="margin-bottom: 7px;">
+      <div class="col-9 col-start-large-4">
+        <nav class="p-tabs" aria-label="Tabs example">
+          <ul class="p-tabs__list">
+            {% for release in release_details["releases"]|sort(attribute='version')|reverse %}
+            <li class="p-tabs__item">
+              <button class="p-tabs__link" role="tab" aria-selected="{{ 'true' if loop.first else 'false' }}" aria-controls="tab{{ loop.index }}" id="tab{{ loop.index }}-button" tabindex="{{ '0' if loop.first else '-1' }}">{{ release.version }}</button>
+            </li>
             {% endfor %}
           </ul>
-        </aside>
+        </nav>      
       </div>
     </div>
-  </section>
-  {# djlint: on #}
+    {% else %}
+    <hr/>
+    {% endif %}
+
+    <div class="row--25-75-on-medium row--25-75-on-large">
+      <div class="col u-hide--small">
+        {{ image(url="https://assets.ubuntu.com/v1/d9c2c5bf-ubuntu_certified.png", alt="Logo of the Ubuntu Certified project", width="112", height="157", hi_def=True, loading="auto") | safe }}
+      </div>
```

Comment:
Thanks for sharing this doc! According to it, I believe the best description for it would be "Ubuntu Hardware Certification" to which I updated the code.